src/Avalonia.Base/Threading/Dispatcher.Queue.cs

```diff
@@ -194,7 +194,6 @@ void ExecuteJobsCore(bool fromExplicitBackgroundProcessingCallback)
                 
                 if (Now - backgroundJobExecutionStartedAt.Value > _maximumInputStarvationTime)
                 {
-                    _signaled = true;
```

Comment:
Unrelated change?

Reply:
Yes. I need to rebase but for some reasons this must wait :) ignore please for now